src/illumidesk/illumidesk/apis/nbgrader_service.py

```diff
@@ -18,24 +16,23 @@
 nbgrader_db_port = os.environ.get("POSTGRES_NBGRADER_PORT") or 5432
 nbgrader_db_password = os.environ.get("POSTGRES_NBGRADER_PASSWORD")
 nbgrader_db_user = os.environ.get("POSTGRES_NBGRADER_USER")
+nbgrader_db_name = os.environ.get("POSTGRES_NBGRADER_DATABASE")
 mnt_root = os.environ.get("ILLUMIDESK_MNT_ROOT", "/illumidesk-courses")
 
 org_name = os.environ.get("ORGANIZATION_NAME") or "my-org"
 
 if not org_name:
     raise EnvironmentError("ORGANIZATION_NAME env-var is not set")
 
+CAMPUS_ID = os.environ.get("CAMPUS_ID")
```

Comment:
if the database does exist in rds instance like in the case of `illumidesk_next_20220201` database, I get the following error

`
Rolling back session due to database error (psycopg2.errors.UndefinedColumn) column course.campus_id does not exist
    LINE 1: ...urse.id AS course_id, course.name AS course_name, course.cam..`